hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -194,16 +187,18 @@ public class HoodieWriteConfig extends HoodieConfig {
 
   public static final ConfigProperty<String> TIMELINE_LAYOUT_VERSION_NUM = ConfigProperty
       .key("hoodie.timeline.layout.version")
-      .defaultValue(Integer.toString(TimelineLayoutVersion.VERSION_1))
+      .defaultValue(Integer.toString(TimelineLayoutVersion.CURR_VERSION))
+      .withValidValues(Integer.toString(TimelineLayoutVersion.VERSION_0),Integer.toString(TimelineLayoutVersion.VERSION_1))
       .sinceVersion("0.5.1")
       .withDocumentation("Controls the layout of the timeline. Version 0 relied on renames, Version 1 (default) models "
           + "the timeline as an immutable log relying only on atomic writes for object storage.");
 
   public static final ConfigProperty<HoodieFileFormat> BASE_FILE_FORMAT = ConfigProperty
       .key("hoodie.table.base.file.format")
       .defaultValue(HoodieFileFormat.PARQUET)
-      .withAlternatives("hoodie.table.ro.file.format")
-      .withDocumentation("Base file format to store all the base file data.");
+      .withValidValues(HoodieFileFormat.PARQUET.name(), HoodieFileFormat.ORC.name(), HoodieFileFormat.HFILE.name())
+      .withDocumentation(HoodieFileFormat.class, "File format to store all the base file data.")
```

Comment:
Same here for using `.withDocumentation(HoodieFileFormat.class)`?

Reply:
That one doesn't work because the enum has HOODIE_LOG as a value